mothernet/tests/test_follow_sklearn_interface.py

```diff
@@ -118,7 +118,7 @@ def test_baam_with_categoricals():
     # Mix Float and categorical features.
     X = np.concatenate([X[:, :2], X_rand_cat[:, 2:]], axis=-1)
     X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
-    model_string = "baam_H512_Dclass_average_e128_nsamples500_numfeatures20_padzerosFalse_03_14_2024_15_03_22_epoch_400.cpkt"
```

Comment:
This doesn't work unless we upload the model, right? can you please test the model prediction as part of the model training tests?